frontend/packages/dev-console/src/components/pipelineruns/PipelineRunVisualization.tsx

```diff
@@ -41,14 +42,21 @@ export class PipelineRunVisualization extends React.Component<
   }
 
   render() {
+    const { pipelineRun } = this.props;
     if (this.state.errorCode === 404) {
       return null;
     }
+    const path = resourcePathFromModel(
+      PipelineRunModel,
+      pipelineRun.metadata.name,
+      pipelineRun.metadata.namespace,
+    );
     return (
       <PipelineVisualizationGraph
-        namespace={this.props.pipelineRun.metadata.namespace}
-        graph={getPipelineTasks(this.state.pipeline, this.props.pipelineRun)}
-        runStatus={pipelineRunFilterReducer(this.props.pipelineRun)}
+        link={path}
```

Comment:
Instead of passing the link, passed the pipeline run name and constructed the link in TaskComponent.
fixed in https://github.com/openshift/console/pull/2695/commits/cbd40192e2245a12ac9f77b8a4dafb079049f312